packages/openapi-code-generator/src/typescript/common/schema-builders/joi-schema-builder.ts

```diff
@@ -118,12 +118,15 @@ export class JoiBuilder extends AbstractSchemaBuilder<
   }
 
   protected object(keys: Record<string, string>): string {
+    const entries = Object.entries(keys)
+
     return [
       joi,
       "object()",
-      `keys({${Object.entries(keys)
+      `keys({${entries
         .map(([key, value]) => `"${key}": ${value}`)
         .join(",")} })`,
+      "options({ stripUnknown: true })",
```

Comment:
Match current `zod` builder behavior and silently strip unknown keys rather than raise an error. This is a personal preference of mine, and so I may introduce a flag to disable / add `.strict()` to the zod schemas, though in my experience this tends to cause more headaches than it solves